webpack/assets/javascripts/react_app/components/Pagination/PaginationHooks.js

```diff
@@ -1,12 +1,9 @@
-import { getURIperPage } from '../../common/urlHelpers';
-import { useForemanSettings } from '../../Root/Context/ForemanContext';
+import { deprecate } from '../../common/DeprecationService';
 
 export const usePaginationOptions = () => {
-  const perPageOptions = new Set([5, 10, 15, 25, 50]);
-  const { perPage } = useForemanSettings();
-  const URIPerPage = getURIperPage();
-
-  perPageOptions.add(perPage);
-  if (URIPerPage) perPageOptions.add(URIPerPage);
-  return [...perPageOptions].sort((a, b) => a - b);
+  deprecate(
+    'usePaginationOptions',
+    3.2,
+    'PF4 pagination which is already shipped with those options'
```

Comment:
This gives:
`DEPRECATION WARNING: you are using deprecated usePaginationOptions, it will be removed in Foreman PF4 pagination which is already shipped with those options. Use 3.2 instead.`
Please swap `3.2` and the instead part :)
```suggestion
    'usePaginationOptions',
    'PF4 pagination which is already shipped with those options',
    3.2
```